mlflow/utils/async_logging/run_batch.py

```diff
@@ -0,0 +1,32 @@
+import threading
+
+from mlflow.entities.metric import Metric
+from mlflow.entities.param import Param
+from mlflow.entities.run_tag import RunTag
+
+
+class RunBatch:
+    def __init__(
+        self,
+        run_id: str,
+        params: [Param],
```

Comment:
`[Param]` is a correct type hint syntax?

Reply:
Updated to have List[Param]